### ISSUE TITLE
Webmin 2.021: Cant change GRPC to "Yes"

### ISSUE THREAD

**Issue by derjohn** [webmin/webmin#1941]
Hey,
via the Webmin GUI i'd like to change the option unter "[Permissions for all modules]" =>  "Can accept RPC calls?" 

from "Only for root or admin" to "yes".

This did work informer Version, if click on "save" the action goes without errors , but the Option doesn't change to "yes".

rgds,
j.

**Comment by derjohn** [comment]
Update: I think I cant change anything, but setting rpc=1 in /etc/webmin/foobar.acl works.

**Comment by jcameron** [comment]
I assume you're trying to make this change for a less-privileged Webmin user?

**Comment by derjohn** [comment]
@jcameron Yes, that what I do. Actually I use Webmin for the Secondary NS, so the user only has permission to "server => bind". If I understood it correctly, that user has full RPC /API access to everything , when I set this to "yes" ?

**Comment by jcameron** [comment]
That is correct - this user will have full access to all APIs, so is equivalent to root.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run.

⚠️ If the issue is resolved, please take a moment to close it to help keep the project organized.
